flow-typed/npm/apollo-link-context_vx.x.x.js

```diff
@@ -0,0 +1,46 @@
+// flow-typed signature: d2e9c275593d777b12757d2dbe083d03
```

Comment:
I wonder if it would be worth hiding these by default, like we do with snapshots?